_summaries/reddi2019on_2.md

```diff
@@ -1,19 +1,61 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{reddi2019on}}
```

Comment:
overall I think the review is very good. I think you cover every important point in the paper, and you mostly use your own words to illustrate the ideas. Just some minor stuff needed to be clarified. Also theorem part is too lengthy for me.

Reply:
@timchen0618 Thanks, I updated my summary following your comments. Let me know if there's anything else you think I should add

Reply:
I think it is great overall! Only one thing about TL;DR, I added a comment to that.